library/CHART-TEMPLATE/Chart.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v2
+name: CHART-TEMPLATE
+description: A Helm chart for Kubernetes
```

Comment:
I would maybe add few lines saying that this chart is just an example to create unit tests for the common library